src/libcore/convert.rs

```diff
@@ -276,7 +276,7 @@ pub trait Into<T>: Sized {
 pub trait From<T>: Sized {
     /// Performs the conversion.
     #[stable(feature = "rust1", since = "1.0.0")]
-    fn from(T) -> Self;
+    fn from(t: T) -> Self;
```

Comment:
IMO `t` is unidiomatic. I personally prefer `x`. Or `_` like you used elsewhere.